src/Android/Renderers/CustomLabelRenderer.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Bit.App.Controls;
+using System.ComponentModel;
+using Xamarin.Forms.Platform.Android;
+using Android.Content;
+using Xamarin.Forms;
+using Bit.Droid.Renderers;
+
+[assembly: ExportRenderer(typeof(CustomLabel), typeof(CustomLabelRenderer))]
+namespace Bit.Droid.Renderers
+{
+    public class CustomLabelRenderer : LabelRenderer
+    {
+        public CustomLabelRenderer(Context context)
+            : base(context)
+        { }
+
+        protected override void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            var label = sender as CustomLabel;
+            switch (e.PropertyName)
+            {
+                case nameof(CustomLabel.AutomationId):
+                    Control.ContentDescription = label.AutomationId;
+                    break;
+            }
+            base.OnElementPropertyChanged(sender, e);
+        }
+    }
```

Comment:
🤔  Does this work to fix the list automation id binding issue? if not, remove it

Reply:
This workaround is valid! It's working because I'm using the `CustomLabel` element instead of `ExtendedStackLayout` (exactly as it was mentioned [here](https://github.com/xamarin/Xamarin.Forms/issues/10492))